README.md

```diff
@@ -1,8 +1,3 @@
-[![Master Build Status](https://ci.centos.org/buildStatus/icon?subject=master&job=devtools-che-devfile-registry-build-master/)](https://ci.centos.org/job/devtools-che-devfile-registry-build-master/)
```

Comment:
could you please use links to github actions instead of removing it ?

Reply:
sure